sematic/utils/db.py

```diff
@@ -0,0 +1,31 @@
+# Standard Library
+import enum
+from typing import Type
+
+# Third-party
+from sqlalchemy.types import Integer, TypeDecorator
+
+
+class IntEnum(TypeDecorator):
```

Comment:
In the long run, we should use something similar for regular enums so that e.g. `run.future_state` is an instance of `FutureState` instead of a string.

Reply:
TIL 😎